WORKING_GROUPS.md

```diff
@@ -129,8 +129,24 @@ The [OpenJS Code of Conduct][] applies to this Working Group.
 
 ## Current Working Groups
 
+* [Code of Conduct](#code-of-conduct)
 * [Standards](#standards)
 
+### [Code of Conduct](https://github.com/openjs-foundation/code-of-conduct/)
+
+The purpose of the Code of Conduct Working Group is to define and maintain a set of resources/processes for use by the Foundation, its teams and its projects.
+
+Responsibilities include:
+
+* Define and maintain overarching Code of Conduct and moderation processes, documentation and resources as they pertain to the Foundation and the Cross Project Council.
+* Define minimum Code of Conduct and moderation requirements for projects as part of Foundation membership and provide guidance/resources to help them implement those requirements.
```

Comment:
I do not expect the mininum requirements to be changed without an approval with the CPC. I think it is important that https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md stays under the purview of the CPC and its reponsibility not to be transfered.

In other terms, I do not see what this working group could add to that document? If we think that should be changed/integrated, I think it should be proposed by the wg and approved by the CPC.

I recommend this to be changed to:

```suggestion
* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md.
```